CoreBluetoothMock/Classes/CBMCentralManagerMock.swift

```diff
@@ -979,7 +990,7 @@ open class CBMPeripheralMock: CBMPeer, CBMPeripheral {
             let initialSize = services!.count
             services = services! + mockServices
                 // Filter all device services that match given list (if set).
-                .filter { serviceUUIDs?.contains($0.uuid) ?? true }
+                .filter { serviceUUIDs?.contains($0.uuid) ?? true || initialSize == 0 }
```

Comment:
Could you change the comment to `Filter all device services that match the given list (if set) or include all of them if the initial list is empty` or something like that?